README.md

```diff
@@ -66,14 +66,29 @@ Screen.poly(20, 50, 20, 100)
 Screen.flush()
 ```
 
-Built-in properties to access the environment, like time or links:
+Built-in properties to access the environment, like counter or time:
 
 ```python
+counter = Env.counter
 current_time = Env.time
-link_count = Env.link_count
 ```
 
-Built-in functions to access sensors, like the amount of copper or current health:
+compiles to:
+
+```
+set counter @counter
+set current_time @time
+```
+
+Beyond defined ones in [`pyndustri.pyi`], this is also the way to write mindustry built-in variables and constants (ones that start with @). If written as string literals, compiled result will be a string, not a variable. E.g:
+
+```python
+titanium = "@titanium" # set titanium "@titanium"
+titanium = @titanium # compile error
+titanium = Env.titanium # set titanium @titanium
```

Comment:
Actually these "built-in" variables are defined for the most part in `Senseable`. Perhaps the `Env` could subclass `Senseable` so that it these type checked?

Reply:
`Senseable` works only as property of something: `a = container1.titanium` -> `sensor a container1 @titanium` 
I needed a way to write builtin variables outside of sensor: `Unit.build(0, 0, Env.sorter, 0, Env.titanium) `
I also think the end user should be ablo to write arbitrarily Mindustry built-ins in some way. First I thought about something like: `built-in("@sorter")` (I tried other compilers) until I figured out that Env goes further than what is written in `pyndustri.pyi` (had to read source code to figure that out, I don't know why it wasn't obvious), had "Ahaaa" moment and decided to write it in docs